src/components/featured-maps-list/featured-maps-list-component.jsx

```diff
@@ -1,25 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cx from 'classnames';
 import { useSpring, animated } from 'react-spring';
-
+import styles from './featured-maps-list-styles.module';
 
 const FeaturedMapsListComponent = ({ 
   className,
   featuredMapsList,
   selectedSidebar,
   isLandscapeMode,
   isFullscreenActive,
-  handleFeaturedMapClick
+  handleFeaturedMapClick,
+  handleFeatureMapHover
 }) => {
   const isOpen = selectedSidebar === 'featuredMapsList';
   const slide = useSpring({
     from: { marginLeft: -400 },
     marginLeft: isOpen && !isLandscapeMode && !isFullscreenActive ? 0 : -400,
     delay: isOpen && !isLandscapeMode && !isFullscreenActive ? 400 : 0
   })
+
+  const defaultCardOpen = featuredMapsList[0];
```

Comment:
If we avoid the `mouseOut` functionality (as recommended on other comment) we will not need this `defaultCardOpen` as the card open will be the `featuredMap` selected (that also would make sense when reaching the `featuredMapsList` from a `featuredMap` as this will be the opened list item).
BTW on the InVision designs the default state is with all the items closed (not sure if we should remove this `defaultCardOpen` anyway).